src/lwc/gsChecklistItem/gsChecklistItem.html

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="step-card">
```

Comment:
Why not using `lightning-card`?

Reply:
@bethbrains  I have problem to creta the design using lightning-card, this card have a image in his left side and is very different to regular lightning-card